_posts/2022-01-14-code-reviews.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Code reviews
```

Comment:
I am not confident with the title. Is it too general? Or is it good to be general?

Alternatives that come to my mind
* Code reviews are important!
* Code reviews are even more important than programming itself
* Motivations and tips for code reviews

Somehow I want to say something specific here, but content of this blog post is also quite general or wide range.